### DISCUSSION TITLE
Download NCBI Genomes by taxonomic id - ncbi-genome-download not working

### DISCUSSION THREAD

**Discussion by nush320** [biostars/biostar-handbook-issues#268] [Q&A]
Hi I downloaded the ncbi-genome-download software as instructed and it seems to be installed when I check using `which ncbi-download-genome`. 
It prints out the following - /home/bstarnush/miniconda3/bin/ncbi-genome-download
When I do the next command - `NCBI-genome-download --genera Streptomyces bacteria --dry-run | head` I get a KeyError: 'assembly-accession'. I have upgraded pip and tried to reinstall again. Same error. 
I also tried the following: `conda install -c bioconda ncbi-genome-download` I got the same error again.

**Comment by ialbert** [comment]
Looks like the same issue as reported here:

https://github.com/kblin/ncbi-genome-download/issues/209

**Reply by nush320** [reply]
Weird I couldn't find that when I googled :( 
Thanks so much!

**Reply by nush320** [reply]
Hi, please see my comment in the same thread I made. Seems to work now but its giving me a "BrokenPipeError" at the end: https://github.com/kblin/ncbi-genome-download/pull/210#issuecomment-1591651829

**Reply by ialbert** [reply]
broken pipe is usually not an error, it means you interrupted the flow out the output

if you redirect into a file it should work fine

     run tool > output_file

**Reply by nush320** [reply]
Didn't quiet understand that. Do you mean I run this: `ncbi-genome-download --genera Streptomyces bacteria --dry-run | head` with a `> output_file`? And if so what type of output file. P.S. I ran the command without `| head` and it printed everything fine.
